### ISSUE TITLE
Add list of dependencies

### ISSUE THREAD

**Issue by wincowgerDEV** [U-Alberta/ADaPT-ML#4]
It would be helpful to add a full list of dependencies. I know that these dependencies are more or less handled within docker but I ran into some issues early on when diving into this tool when I realized the best way to use this tool is with a Linux machine. If there was a dependencies file in the documentation and linked early on in the README that would make sure people know what they need to do before they get into the tool.

**Comment by nulberry** [comment]
I've changed a couple things in commit 9412302be59a109bfbd5e2a230a83ca0fe45f710 to address this:
- Step 1 of usage.md now explicitly states required and recommended system setup. 
- For required setup, I have included the most compatible versions of Docker and Docker Compose. I have also noted the approximate amount of space that the Docker images take up, which will hopefully prevent users from having a similar issue with disk space that you had in #7.
- For recommended setup, I have noted the Linux distribution that the automated tests run on. I have also moved up the CrateDB bootstrap checks from the Additional Installation Notes section at the bottom because these steps seem to be more essential than I thought; I needed to perform these checks when I first started working on this project, and I think that these checks will address the CrateDB container restarting problem in #9.

I hope the added clarity helps.
